github/checks_test.go

```diff
@@ -467,6 +467,7 @@ func TestChecksService_RequestCheckSuite(t *testing.T) {
 	mux.HandleFunc("/repos/o/r/check-suite-requests", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "POST")
 		testHeader(t, r, "Accept", mediaTypeCheckRunsPreview)
+		testBody(t, r, "{\"head_sha\":\"deadbeef\"}"+"\n")
```

Comment:
Sorry that I didn't see this before, but we can simplify this line and make it easier to read:
```
testBody(t, r, `{"head_sha":"deadbeef"}`+"\n")
```

Reply:
Great suggestion! Applied in f9cebb7.